### PR TITLE
Add the v7 module to dependencies

### PR DESCRIPTION
**Pull request by nfurfaro**
# Description
All in the title.

# Issues

Fixes #
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: smart-contracts/package.json [hunk 1]
```diff
@@ -59,7 +59,8 @@
     "unlock-abi-1-0": "1.0.0",
     "unlock-abi-1-1": "1.1.1",
     "unlock-abi-1-2": "1.0.4",
-    "unlock-abi-1-3": "1.0.2"
+    "unlock-abi-1-3": "1.0.2",
+    "unlock-abi-7": "^1.0.2"
```
Comment (NickCuso) [inline smart-contracts/package.json:63]: Let's pin the dependency like the others

Comment (nfurfaro) [reply smart-contracts/package.json]: yup. will do.

**Comment by NickCuso** [comment]
hmm that didn't change the lock file too?

**Comment by julien51** [comment]
@nfurfaro You might want to run `yarn` inside of the folder.

**Pull request merged**